prime-router/src/main/kotlin/cli/tests/Hl7Tests.kt

```diff
@@ -92,7 +91,7 @@ class Hl7Ingest : CoolTest() {
 class BadHl7 : CoolTest() {
     override val name = "badhl7"
     override val description = "Submit bad hl7 scenarios - should get errors"
-    override val status = TestStatus.SMOKE
+    override val status = TestStatus.DRAFT // This test fails or hangs sometimes when part of the smokes.  Not sure why
```

Comment:
Open a new issue to get this resolved.  Which environment is this on?  I know the super large payload behaves differently depending if it is local or not

Reply:
See #2265